handbook/communication/content_guidelines/terminology_guidelines.md

```diff
@@ -6,10 +6,10 @@ We use certain words in a special way. This word list defines our terminology to
 
 We describe ourselves with a few different names depending on context, and we should use the right term at the right time.
 
-- **Sourcegraph**: Main product. Prefer using this name unless you need to be more precise.
-- **Self-hosted Sourcegraph instance**: Only if clarification between Sourcegraph Cloud, managed instances, and on-premises instances is required.
-- **Managed Sourcegraph instance**: Only if clarification between the Sourcegraph Cloud instance, self-hosted instances, and managed instances is required.
-- **Sourcegraph Cloud**: The public instance of Sourcegraph for open source code at [sourcegraph.com](https://sourcegraph.com).
+- **Sourcegraph**: Main product. This name is always preferred unless you need to clarify between the 3 deployment methods for Sourcegraph below.
+- **Sourcegraph self-hosted**: On-prem and self-managed version of Sourcegraph. 
```

Comment:
```suggestion
- **Sourcegraph self-hosted**: On-premises and self-managed version of Sourcegraph. 
```

Changed to the full term here to avoid abbreviations (and was pleasantly surprised to learn that [the singular term is "premises," not "premise"](https://www.govloop.com/on-premise-vs-on-premises-the-debate-and-resolution/)).